### PR TITLE
2019-03-25-CoinBene

### PR DESCRIPTION
**Pull request by JediFaust**

### CODE REVIEW

File: content/attacks/posts/2019-03-25-CoinBene.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+The identity of the attacker remains unknown. The following addresses were involved in the hack:
+
+- [0xA1BF1ED1e8DE34477fB3dCE27c2eA2EA4163AcbA](https://etherscan.io/address/0xa1bf1ed1e8de34477fb3dce27c2ea2ea4163acba)
+- [0xb3dF999C5dC026dea265AEB02b8519844C9b6D5E](https://etherscan.io/address/0xb3dF999C5dC026dea265AEB02b8519844C9b6D5E)
+- [0xba351e7f0c630b3baa30a0ff38f6f4a333ef2133](https://etherscan.io/address/0xba351e7f0c630b3baa30a0ff38f6f4a333ef2133)
+
+## Losses
+
+CoinBene lost $105,150,329 worth of various assets. The most valuable [stolen assets breakdown](https://github.com/elementus-io/coinbene-analysis#market-value-of-assets-withdrawn):
```
Comment (jhirschkorn) [inline content/attacks/posts/2019-03-25-CoinBene.md:25]: The second sentence here is a little awkward.  Suggest: "Breakdown of top 5 stolen assets by value:"

File: content/attacks/posts/2019-03-25-CoinBene.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+- $70,738,770 worth of Maximine Coin
+- $14,252,973 worth of CoinBene Coin
+- $4,744,863 worth of Guaranteed Ethurance Token
+- $2,693,476 worth of Huobi Pool Token
+- $2,575,313 worth of Ethereum
+
+## Timeline
+
+- **March 20, 2019:** [Bitwise's presentation to the SEC](https://www.sec.gov/comments/sr-nysearca-2019-01/srnysearca201901-5164833-183434.pdf) mentioned CoinBene, with the suspicions of wash trading.
```
Comment (jhirschkorn) [inline content/attacks/posts/2019-03-25-CoinBene.md:35]: ...mentioned suspicions of wash trading on CoinBene

File: content/attacks/posts/2019-03-25-CoinBene.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- **March 26, 2019, 06:20 AM UTC:** CoinBene [announced that the platform was undergoing maintenance](http://web.archive.org/web/20200401054024/https://twitter.com/CoinBene/status/1110531859539070977) due to upgrading the platform wallet. The tweet was deleted later.
+- **March 26, 2019, 01:29 PM UTC:** Crypto expert Nick Schteringard [said on Twitter that the exchange's users reported their wallets being hacked](https://twitter.com/schteringard/status/1110534136698159110) and provided [the attacker's address](https://etherscan.io/address/0x6585329751de1140d68bd6cad1b46ebec1131f75).
+- **March 27, 2019:** Data scientists at Elementus [issued a report suggesting that the fund transfers out of CoinBene's hot wallet reached approximately $105 million](https://www.elementus.io/blog-post/coinbene-analysis).
+- **April 9, 2019:** PeckShield, a blockchain security company, [published a report](https://peckshield.medium.com/coinbene-incident-investigation-report-6d4a3ed3b715) stating that the stolen funds were transferred additionally to the centralized exchanges, including Binance, Bittrex, Huobi, and OKEx.
```
Comment (jhirschkorn) [inline content/attacks/posts/2019-03-25-CoinBene.md:41]: ...funds were transferred to centralized exchanges,...

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

**Comment by jhirschkorn** [comment]
@JediFaust are you still workkng this one?

**Comment by JediFaust** [comment]
> @JediFaust are you still workkng this one?

Yeah, but no need to merge yet, will debug grammar checker on this one

**Review by jhirschkorn — APPROVED** [review]

**Pull request merged**